consent-ws/src/test/java/org/broadinstitute/consent/http/NihAccountTest.java

```diff
@@ -0,0 +1,98 @@
+package org.broadinstitute.consent.http;
+
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import org.broadinstitute.consent.http.configurations.ConsentConfiguration;
+import org.broadinstitute.consent.http.enumeration.ResearcherFields;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+
+public class NihAccountTest extends AbstractTest{
```

Comment:
I don't have a suggestion, but would love to see if there's a way to test a security scenario. Will try to spend some time digesting the OWASP doc and see if there's something more we can look for.